added possibility to retrieve hcloud cluster config from file (#7817)
kubernetes/autoscaler: cluster-autoscaler/cloudprovider/hetzner/README.md

}
```

`HCLOUD_CLUSTER_CONFIG_FILE` Can be used as alternative to `HCLOUD_CLUSTER_CONFIG`. This is the path to a file
containing the JSON structure described above. The file will be read and the contents will be used as the configuration.
Can be useful when you have many different node pools and run into issues of the env var becoming too long.

> **NOTE**: In contrast to `HCLOUD_CLUSTER_CONFIG`, this file will not be base64 encoded.

Reviewer: ```suggestion
**NOTE**: In contrast to `HCLOUD_CLUSTER_CONFIG`, this file is not base64 encoded.
```
